proof_generation/helper/co_gateway.js

```diff
@@ -0,0 +1,441 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const EventDecoder = require('../../test/test_lib/event_decoder');
+
+/**
+ * Object contains input parameter for confirmStakingIntent.
+ *
+ * @typedef {Object} Params
+ * @property {string} params.staker Staker address.
```

Comment:
```suggestion
 * @property {string} staker Staker address.
```
Similar comment for remaining parameters.